index.html

```diff
@@ -5,16 +5,83 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <link rel="stylesheet" type="text/css" href="css/grid.css">
+    <link rel="stylesheet" type="text/css" href="css/grid.css"> 
+    <link rel="stylesheet" type="text/css" href="css/main.css">
     <!-- Add a link to your custom CSS here -->
 
     <title>Responsive grid project</title>
 </head>
-
 <body>
+<div class="container">
+    <div class="row"> <!--header-->
+        <header>
+            <div class="col-6">
+            <h1 class="logo-title">ALLAN JOHNSON</h1>
+            <h3 class="logo-sub">Personal Life Coach</h3>
+        </div>
+            <nav>
+                <div class="col-6">
```

Comment:
Similar comment as above - move the `col-...` class to be directly on the `nav` so that it is a direct child of the `row`. And get rid of the div on line 23 as you won't need it anymore.